### PR TITLE
[Communication] - SMS - Enabling SMS Live tests using Static Resources

### PR DESCRIPTION
**Pull request by minnieliu**

### CODE REVIEW

File: sdk/communication/azure-communication-sms/pom.xml [hunk 1]
```diff
@@ -52,10 +52,6 @@
         <artifactId>jacoco-maven-plugin</artifactId>
         <version>0.8.5</version> <!-- {x-version-update;org.jacoco:jacoco-maven-plugin;external_dependency} -->
         <configuration>
-          <properties>
```
Comment (srnagar) [inline sdk/communication/azure-communication-sms/pom.xml:55]: This whole jacoco plugin can be removed.

**Review by srnagar — APPROVED** [review]

File: sdk/communication/azure-communication-sms/pom.xml (hunk 1 above)
Comment (minnieliu) [reply sdk/communication/azure-communication-sms/pom.xml:55]: Addressed, thank you!

**Review by turalf — APPROVED** [review]

**Pull request merged**